.github/workflows/main.yml

```diff
@@ -3,19 +3,36 @@ name: Main
 on: [push, pull_request]
 
 jobs:
+
   lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Setup Python
-        uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
         with:
           python-version: 3.8
-      - name: Install tox
-        run: python -m pip install tox
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip setuptools wheel tox
       - name: Run linting
         run: python -m tox -e lint
 
+  types:
```

Comment:
Do we want to make the `types` and `docs` jobs ["Required" checks](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/enabling-required-status-checks) (along with `lint` and `test`)? They were implicitly required by Travis.

If so, I don't have permission to do that. Which of the @pypa/twine-maintainers does?

Reply:
I think they need to run on master before they can be made required?